### ISSUE TITLE
window.open signature has a replace parameter that the docs don't have

### ISSUE THREAD

**Issue by pushkin-** [microsoft/TypeScript#31246]
**TypeScript Version:**  3.4.3

**Search Terms:** window open replace

**Code** `window.open(`

Intellisense shows a `replace` property found here in lib.dom.d.ts:

    open(url?: string, target?: string, features?: string, replace?: boolean): Window | null;

However, the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Parameters) don't have such a parameter.

I did see that parameter listed [here](https://www.w3schools.com/jsref/met_win_open.asp) on W3Schools, but that's not a trustworthy source. Maybe it's an old parameter that was removed.

**Comment by fatcerberus** [comment]
MSDN lists the `replace` parameter:
https://msdn.microsoft.com/en-us/ie/ms536651(v=vs.94)

So it's either IE-specific, or else somehing that existed in a previous standard and is no longer officially a thing.

**Comment by pushkin-** [comment]
@fatcerberus Hmm found [another page](http://help.dottoro.com/ljuhanvh.php) that lists that parameter and indicates that it's supported by Chrome and FF...

**Comment by pushkin-** [comment]
At least in Chrome, this doesn't appear to have an effect.
```
window.open("https://google.com", "a"); // opens new tab to google
window.open("https://google.com", "a"); // navigates that tab to google
// press and hold the tab's back button to see the history
window.open("https://google.com", "a", "", true); // navigates the tab to google again
// the history is still updated with an entry even though the w3schools page suggests that the new page should replace the existing page in history.
```

**Comment by scottt13** [comment]
This has popped its head up for me recently. Chrome dropped support for the 4th parameter in version 89 and using it now causes an error.

MDN does not mention the replace parameter at all:
https://developer.mozilla.org/en-US/docs/Web/API/Window/open

**Comment by azack** [comment]
Reference for the Chrome change https://chromium.googlesource.com/chromium/src/+/0ed07dc3f70fb426b64f886fc2f0018243443f9e

Related bug and discussion:
https://bugs.chromium.org/p/chromium/issues/detail?id=1164959

**Comment by jakebailey** [comment]
Just looking at old issues; this was fixed in #44684.
